collect_app/src/main/res/layout/main_menu.xml

```diff
@@ -89,13 +89,22 @@
                     android:layout_marginTop="@dimen/margin_extra_small"
                     tools:text="@string/manage_files" />
 
+                <TextView
+                    android:id="@+id/app_name"
+                    style="@style/TextAppearance.Collect.Body1"
+                    android:layout_width="wrap_content"
+                    android:layout_height="wrap_content"
+                    android:layout_gravity="center_horizontal"
+                    android:layout_marginTop="@dimen/margin_standard"
+                    android:textColor="@color/dark_gray"
```

Comment:
No problem with this being gray but it needs to use theme colors rather than an explicit colors so it changes with theme. I think this should probably be `@color/color_on_surface_medium_emphasis` (which is just applies an alpha to `?colorOnSurface`). There is also low and high emphasis versions if that doesn't look right.

Reply:
Great, thanks. I had that question at #4601 because I was not finding the "color role" I wanted.

Reply:
Ah right I hadn't had a chance to look at the PR. It's very likely that new blue you've introduced should be our "primary" color. I've added a note to try and kill `colors.xml` as part of https://github.com/getodk/collect/issues/4572.